sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -64,6 +77,21 @@ object Klaviyo {
         } ?: throw LifecycleException()
 
         UserInfo.startObservers()
+
+        Registry.get<ApiClient>().startService()
+
+        if (preInitQueue.isNotEmpty()) {
+            val operationCount = preInitQueue.count()
+            val operation = if (operationCount == 1) "operation" else "operations"
+
```

Comment:
since this is just for logging, no need to be fancy with the message, can simply do `operation(s)` and skip the singular check

Reply:
fiiiine